fmn/api/handlers/misc.py

```diff
@@ -80,8 +86,7 @@ async def get_artifacts(
         },
     ]
     namespaces = [at.value for at in ArtifactType]
-    projects = await distgit_client.get_projects(pattern=name)
-    for project in projects:
+    for project in await distgit_client.get_projects(pattern=f"*{name}*"):
```

Comment:
Are you sure we need to do a substring search by padding asterisks? It feels like we may end up with more projects than we expect.

Reply:
This was done in the (`fmn.rules`) distgit client previously, I moved that out of the implementation in `fmn.backends`. I'm fine with removing it altogether or move this detail into the frontend (people will probably want to search by substring, I know I want to).